All_years/04-Social_Care/04-SDS_data.R

```diff
@@ -0,0 +1,167 @@
+#####################################################
+# Social Care SDS Data
+# Author: Jennifer Thom
+# Date: July 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - Data from Social care database DVPROD
+# Description - Process SDS data
+#####################################################
+
+# Load packages
+
+library(readr)
+library(dplyr)
+library(dbplyr)
+library(phsmethods)
+library(tidyverse)
```

Comment:
Probably don't need to load tidyverse since we're loading other packages specifically.
```suggestion
```